### ISSUE TITLE
Mono source build always builds both iOS and Mac 

### ISSUE THREAD

**Issue by filipnavara** [xamarin/xamarin-macios#7427]
Compiling with `--disable-ios` still builds all the Mono iOS targets if the Mono source build is used instead of consuming SDK binaries.

It's something that would be nice to fix but aside from longer build times doesn't have a big impact.

**Comment by chamons** [comment]
@rolfbjarne ?

**Comment by rolfbjarne** [comment]
Yes, this is correct, but not a big deal, especially since we're using the mono archive now and build times aren't affected as much.

In any case we accept pull requests if this is something someone wants to try to take a stab at.

**Comment by rolfbjarne** [comment]
This won't be an issue once we're only building for .NET.
